pom.xml

```diff
@@ -138,16 +138,26 @@
         <artifactId>atlassian-plugins-core</artifactId>
         <version>4.0.0-m029</version>
       </dependency>
-      <dependency>
-        <groupId>org.jenkins-ci</groupId>
-        <artifactId>annotation-indexer</artifactId>
-        <version>1.13</version>
-      </dependency>
       <dependency>
         <groupId>org.codehaus.plexus</groupId>
         <artifactId>plexus-utils</artifactId>
         <version>3.1.0</version>
       </dependency>
+      <dependency>
+        <groupId>com.infradna.tool</groupId>
+        <artifactId>bridge-method-annotation</artifactId>
+        <version>1.23</version>
+      </dependency>
+      <dependency>
```

Comment:
matrix project is part of the bom - either we need a newer bom or we should not be using the bom at all.

Reply:
```
[WARNING] Rule 5: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for org.jenkins-ci.plugins:matrix-project:1.19 paths to dependency are:
+-org.jenkins-ci.plugins.to-declarative:declarative-pipeline-migration-assistant:1.5.3-SNAPSHOT
  +-org.6wind.jenkins:lockable-resources:2.14
    +-org.jenkins-ci.plugins:matrix-project:1.19 (managed) <-- org.jenkins-ci.plugins:matrix-project:1.18
and
+-org.jenkins-ci.plugins.to-declarative:declarative-pipeline-migration-assistant:1.5.3-SNAPSHOT
  +-org.jenkins-ci.plugins:htmlpublisher:1.29
    +-org.jenkins-ci.plugins:matrix-project:1.19 (managed) <-- org.jenkins-ci.plugins:matrix-project:1.20
and
+-org.jenkins-ci.plugins.to-declarative:declarative-pipeline-migration-assistant:1.5.3-SNAPSHOT
  +-org.jenkins-ci.plugins:jira:3.6 [test]
    +-org.jenkins-ci.plugins:matrix-project:1.19 [test] (managed) <-- org.jenkins-ci.plugins:matrix-project:1.18 [test]
```

Reply:
again included in `1135.va_4eeca_ea_21c1` (possibly older version too)

Reply:
see https://github.com/jenkinsci/declarative-pipeline-migration-assistant-plugin/pull/134